requirements/base.txt

```diff
@@ -7,3 +7,4 @@ future>=0.17.1  # required for builtins in setup.py
 # pyyaml>=3.13
 PyYAML>=5.1  # OmegaConf requirement >=5.1
 tqdm>=4.41.0
+fsspec>=0.8.0
```

Comment:
is this a minimal or this was just you are using?

Reply:
Ah ya, that was just the version I tested on. I think this will work for my purposes as early as `0.6`. We can relax this.

Reply:
I will check it in #3132